akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -315,8 +312,15 @@ private[stream] final class GraphInterpreterShell(
   private var self: ActorRef = _
   lazy val log = Logging(mat.system.eventStream, self)
 
-  lazy val interpreter = new GraphInterpreter(assembly, mat, log, inHandlers, outHandlers, logics,
-    (logic, event, handler) ⇒ self ! AsyncInput(this, logic, event, handler), settings.fuzzingMode)
+  private var canEnqueueToShortCircuit: (AsyncInput) ⇒ Boolean = _
```

Comment:
Another issue, though: the purpose of this function is to actually enqueue if possible, which is a bit misrepresented in the name here. I would suggest `hasEnqueuedToShortCircuit`.